symbolic-base/test/Tests/ReedSolomon.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+module Tests.ReedSolomon where
+
+import           Data.Bool                                  (bool)
+import qualified Data.Vector                                as V
+import           GHC.Natural                                (Natural)
+import           Prelude
+import qualified Prelude                                    as P
+import           Test.Hspec
+import           Test.QuickCheck
+
+import qualified ZkFold.Base.Algebra.Basic.Class            as C
+import           ZkFold.Base.Algebra.Basic.Class            hiding ((*), (+))
+import           ZkFold.Base.Algebra.Basic.Field
+import           ZkFold.Base.Algebra.Polynomials.Univariate
+import           ZkFold.Base.Algorithm.ReedSolomon
+
+data ReedSolomonExample f = ReedSolomonExample
+    {
+        char         :: Natural,  -- field order
```

Comment:
field order or field characteristic?
They're the same iff the field has prime order.